source/candl.c

```diff
@@ -47,10 +47,9 @@
 #include <candl/util.h>
 #include <candl/piplib.h>
 
-
 int main(int argc, char * argv[]) {
   
-  osl_scop_p scop = NULL;
+  osl_scop_p scop = NULL, s1, s2;
```

Comment:
Ok. I generally always initialize all variable even if there is not path were they can be used uninitialized, just to be sure. But everybody do it its way.